ext/spl/spl_fixedarray.stub.php

```diff
@@ -24,24 +24,20 @@ public function getSize() {}
     /** @return bool */
     public function setSize(int $size) {}
 
-    /**
-     * @param mixed $index
-     * @return bool
-     */
-    public function offsetExists($index) {}
+    /** @return bool */
+    public function offsetExists(mixed $index) {}
```

Comment:
I think this one should also be `@param int` like the others.

Reply:
Fixed this one too in a new commit. I had to force push again, because there was a conflict in `TIDY_APPLY_CONFIG`.